content/modules/add-module-to-cluster.md

```diff
@@ -70,34 +70,58 @@ To upgrade a module that is installed on a cluster:
 
     ![module_info-1](/images/rs/module_info-1.png?width=1000&height=382)
 
-1. To upgrade the module for the database, run:
+1. To see the versions of the modules on the cluster, run either:
+
+    - `rladmin status modules` - Shows the latest modules available on the cluster
+        and the modules used by databases.
+    - `rladmin status modules all` - Shows all of the modules available on the cluster
+        and the modules used by databases.
+
+1. To upgrade modules for the database, run:
 
     ```sh
-    rladmin upgrade db <database_name> and module module_name <module_name> version <new_module_version_number> module_args "<module arguments>"
+    rladmin upgrade db <database_name | database_ID> and module module_name <module_name> version <new_module_version_number> module_args "<module arguments>"
```

Comment:
I know database_id works. Has database_name been verified with R&D?

Reply:
Verified with Guy and on live system